lua/chainsaw/init.lua

```diff
@@ -120,5 +120,22 @@ function M.removeLogs()
 	vim.b["timelogStart"] = nil
 end
 
+--------------------------------------------------------------------------------
+
+M.commandTable = {
+  assertLog = M.assertLog,
+  beepLog = M.beepLog,
+  debugLog = M.debugLog,
+  messageLog = M.messageLog,
+  objectLog = M.objectLog,
+  removeLog = M.removeLogs,
+  stackTraceLog = M.stracktraceLog,
```

Comment:
here you still got the "stracktrace" typo 😛

Reply:
pulled from main